typescript/widgets/src/walletIntegrations/ethereum.ts

```diff
@@ -101,6 +102,44 @@ export function useEthereumTransactionFns(
     },
     [config, multiProvider],
   );
+
+  return { onSwitchNetwork };
```

Comment:
This returned function is an action, not an event handler. A better action-oriented name would be `switchNetwork`. This also aligns with `addAsset` and other places like `ChainTransactionFns`.